src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+public class Rectangle {
+    private Point topLeftPoint;
+    private Point bottomRightPoint;
+
+    public Rectangle(Point topLeftPoint, Point bottomRightPoint) {
+        this.topLeftPoint = topLeftPoint;
+        this.bottomRightPoint = bottomRightPoint;
+    }
+
+    public Point getTopLeftPoint() {
+        return topLeftPoint;
+    }
+
+    public void setTopLeftPoint(Point topLeftPoint) {
+        this.topLeftPoint = topLeftPoint;
+    }
+
+    public Point getBottomRightPoint() {
+        return bottomRightPoint;
+    }
+
+    public void setBottomRightPoint(Point bottomRightPoint) {
+        this.bottomRightPoint = bottomRightPoint;
+    }
```

Comment:
"Violation: Getters and setters should be placed at the very end
Solution: Move getters and setters to the end"